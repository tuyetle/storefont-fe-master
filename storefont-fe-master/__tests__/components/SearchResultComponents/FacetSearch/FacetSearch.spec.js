import React from 'react';
import MTFilterForm from '~/components/SearchResultComponents/FacetSearch/FacetSearch';
import renderer from 'react-test-renderer';

describe('(Component) MTFilterForm', () => {
  let component;
  let props;

  beforeEach(() => {
    props = {
      items: [],
      onSelectFacetSearchItem() {},
    };
    component = <MTFilterForm {...props} />;
  });

  it('should render conrectly', () => {
    const tree = renderer.create(component).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
