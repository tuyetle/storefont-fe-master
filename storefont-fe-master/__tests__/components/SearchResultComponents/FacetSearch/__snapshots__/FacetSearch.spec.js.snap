// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) MTFilterForm should render conrectly 1`] = `
<nav>
  <ul
    className={
      Object {
        "data-css-1ip94nh": "",
      }
    }
  />
</nav>
`;
